lib/api/uikit/class-beans-uikit.php

```diff
@@ -119,6 +122,21 @@ public function register_less_components() {
 	 * @return array
 	 */
 	public function register_js_components() {
+		return $this->get_registered_component_paths( array( 'core', 'utility', 'touch' ), false );
+	}
+
+	/**
+	 * Gets an array of registered component paths, i.e. absolute path to each component file.
```

Comment:
We are using `Get` 84 times in 40 files and `Gets` 2 times in 2 files. 
I'd suggest continuing with `Get` and changing the other two files.
(this applies to all the `Gets` in this file.)